spec/preservation/create_moab_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe Preservation::CreateMoab do
+  context '#create_moab' do
+    let(:mock_so) { instance_double(Moab::StorageObject) }
+    let(:mock_new_version) { instance_double(Moab::StorageObjectVersion) }
+    let(:verification_result) { instance_double(Moab::VerificationResult) }
+
+    it 'returns true if version manifests match the actual files' do
```

Comment:
I have confirmed that the "perform" step does not need to (and should not?) return "true" by chasing through robot-controller code, etc.  Returning "true" is meaningless in this context.  :-P

So this test description might be more like:  "verifies new version of moab"


I *thought* that we couldn't use mock objects in let statements - delighted to be wrong ... if I am wrong (i.e.  does it always work regardless of expectations?)